pkgs/development/python-modules/linien-common/tests.nix

```diff
@@ -1,26 +1,44 @@
 {
   buildPythonPackage,
-  linien-common,
+  cma,
   linien-client,
+  linien-common,
+  matplotlib,
+  migen,
+  misoc,
+  pytest-plt,
   pytestCheckHook,
 }:
 
 buildPythonPackage {
   pname = "linien-tests";
   inherit (linien-common) version src;
-  format = "other";
   pyproject = false;
 
   dontBuild = true;
   dontInstall = true;
 
   nativeCheckInputs = [
-    linien-common
+    cma
     linien-client
+    linien-common
```

Comment:
it works since `linien-common` in turn don't depend on the test. `pkgs` is a lazy-eval fix point, which is how this works

Reply:
> it works since `linien-common` in turn don't depend on the test. `pkgs` is a lazy-eval fix point, which is how this works

Oh I understand, I haven't noticed this the `tests.nix` file.